config/fund_loader_config/cyp/cyp_r1.py

```diff
@@ -0,0 +1,165 @@
+    },
+    {
+        "section_name": {"en": "4.3 Project milestones", "cy": ""},
+        "form_name_json": {"en": "project-milestones-cyp", "cy": " "},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4.3",
+    },
+    {
+        "section_name": {"en": "4.4 Objectives and activites", "cy": ""},
+        "form_name_json": {"en": "objectives-and-activities-cyp", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4.4",
+    },
+    {
+        "section_name": {"en": "4.5 Location of activites", "cy": ""},
+        "form_name_json": {"en": "location-of-activities-cyp", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4.5",
+    },
+    {
+        "section_name": {"en": "4.6 Working with fund beneficiaries", "cy": ""},
+        "form_name_json": {"en": "working-with-fund-beneficiaries-cyp", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4.6",
+    },
+    {
+        "section_name": {"en": "5. Risk and deliverability", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.5",
+    },
+    {
+        "section_name": {"en": "5.1 Risk and deliverability", "cy": ""},
+        "form_name_json": {"en": "risk-and-deliverability-cyp", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.5.1",
+    },
+    {
+        "section_name": {"en": "6. Value for money", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.6",
+    },
+    {
+        "section_name": {"en": "6.1 Value for money", "cy": ""},
+        "form_name_json": {"en": "value-for-money-cyp", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.6.1",
+    },
+    {
+        "section_name": {"en": "7. Declarations", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.7",
+    },
+    {
+        "section_name": {"en": "7.1 Declarations", "cy": ""},
+        "form_name_json": {"en": "declarations-cyp", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.7.1",
+    },
+]
+
+fund_config = {
+    "id": CYP_FUND_ID,
+    "name_json": {
+        "en": "Children and Young People Resettlement Fund",
+        "cy": "",
+    },
+    "title_json": {
+        "en": "funding to resettle children and young people in England",
+        "cy": "",
+    },
+    "short_name": "CYP",
+    "description_json": {"en": "", "cy": ""},
+    "welsh_available": False,
+    "owner_organisation_name": "Department for Levelling Up, Housing and Communities",
+    "owner_organisation_shortname": "DLUHC",
+    "owner_organisation_logo_uri": DLUHC_LOGO_PNG,
+}
+
+round_config = [
+    {
+        "id": CYP_ROUND_1_ID,
+        "fund_id": CYP_FUND_ID,
+        "title_json": {"en": "Round 1", "cy": ""},
+        "short_name": "R1",
+        "opens": CYP_R1_OPENS_DATE,
+        "deadline": CYP_R1_DEADLINE_DATE,
+        "assessment_deadline": CYP_R1_ASSESSMENT_DEADLINE_DATE,
+        "prospectus": CYP_PROSPECTS_LINK,
+        "privacy_notice": "https://www.gov.uk/guidance/night-shelter-transformation-fund-2022-2025-privacy-notice",
```

Comment:
This privacy notice looks wrong, looks to be night shelters one?

Reply:
empty string for now, will add in a seperate PR when we have this available 👍🏾